docs/getting-started/deploy-singlenode-cluster.md

```diff
@@ -0,0 +1,103 @@
+---
+
+A three-node [Harvester cluster](../glossary/#harvester-cluster) is required to fully realize multi-node features such as high availability. Certain versions of Harvester allow you to create clusters with two management nodes and one [witness node](../advanced/witness/) (and optionally, one or more worker nodes). You can also create [single-node clusters](../advanced/singlenodeclusters/) that support most Harvester features (excluding high availability, multi-replica support, and live migration). 
+
+This guide walks you through the steps required to deploy a **single-node cluster** and virtual machines (VMs) that can host [guest clusters](../glossary/#guest-cluster--guest-kubernetes-cluster). 
+
+## 1. Verify that the minimum hardware and network requirements are met. 
+
+Harvester is built for bare metal servers using enterprise-grade open-source software components. The installer automatically checks the hardware and displays warning messages if the minimum [requirements](../install/requirements) are not met. 
+
+## 2. Prepare the installation files based on the installation method that you want to use. 
+
+You can download the installation files from the [Harvester Releases](https://github.com/harvester/harvester/releases) page. The **Downloads** section of the release notes contains links to the ISO files and related artifacts. The following types of ISO files are available: 
+
+- **Full ISO**: Contains the core operating system components and all required container images, which are preloaded during installation. You must use a full ISO when installing Harvester behind a firewall or proxy, and in environments without internet connectivity. 
+- [**Net install ISO**](../install/net-install): Contains only the core operating system components. After installation is completed, the operating system pulls all required container images from the internet (mostly from Docker Hub). 
+
+
+| Method | Required Installation Files | Other Requirements |
+| --- | --- | --- |
+| [ISO](../install/index) | ISO | N/A |
+| [USB](../install/usb-install) | ISO | USB flash drive; utility such as [balenaEtcher](https://etcher.balena.io/) or the Linux [dd command](https://man7.org/linux/man-pages/man1/dd.1.html) |
+| [PXE](../install/pxe-boot-install) | ISO, Linux kernel image (vmlinuz), initrd, SquashFS image | Directory on the HTTP server (for serving boot files); iPXE boot scripts (for automatic installation); DHCP server configuration |
+
+## 3. Prepare the cluster configuration requirements. 
+
+- Cluster token: ASCII string that nodes use when joining the cluster 
+- Fixed IP address for each node: May be assigned statically or using DHCP (host reservation) 
+- Fixed virtual IP address (VIP) to be used as the cluster management address: VIP that you connect to when performing administration tasks after the cluster is deployed 
+- Addresses of DNS servers, NTP servers, and the proxy server (if necessary) 
+
+## 4. Deploy the cluster node. 
```

Comment:
Suggest to `Install the cluster`, https://docs.harvesterhci.io/v1.4/install/index

Reply:
In your example, the verb "install" applies to the Harvester OS and required packages, which is correct because "install" describes adding software and hardware to a device. For clusters (regardless of the type), the verbs commonly used are "deploy" and "create". We should always try to use terms that best communicate the concepts being described (instead of using some arbitrary differentiation). This suggestion is rejected.